packages/integration_test/README.md

```diff
@@ -7,78 +7,89 @@ and native Android instrumentation testing.
 ## Usage
 
 Add a dependency on the `integration_test` and `flutter_test` package in the
-`dev_dependencies` section of pubspec.yaml. For plugins, do this in the
-pubspec.yaml of the example app.
+`dev_dependencies` section of `pubspec.yaml`. For plugins, do this in the
+`pubspec.yaml` of the example app.
 
-Invoke `IntegrationTestWidgetsFlutterBinding.ensureInitialized()` at the start
-of a test file, e.g.
+Create a `integration_test/` directory for your package. In this directory,
```

Comment:
In the cirrus file we are still using test_driver/ directory even though it is not the recommended directory?

Reply:
Oh sorry, I just read the recommended package structure.

Is there a reason for these recommendations? One idea that comes to my mind is: integration_test_driver.dart won't be the same for different tests.

Reply:
It won't be the same but I would think for most use cases the bulk of the testing code should go into the test script that runs on the device, and the driver is mainly for reporting.

Currently, some packages place the test scripts in `test/foo_e2e.dart` or `test_driver/foo_e2e.dart`. This works today, because of the old e2e naming, but if they were to rename their tests to like `test/foo_integration_test.dart` and  `test_driver/foo_integration_test.dart`, this would conflict with globbing rules today that check for the `*_test.dart` suffix, hence the recommendation of a new `integration_test/` directory for such tests

Reply:
We did the renaming like this last week in the for the web engine: https://github.com/flutter/engine/pull/20954